include/dpp/channel.h

```diff
@@ -185,26 +256,30 @@ class DPP_EXPORT channel : public managed, public json_interface<channel>  {
 	/** Permission overwrites to apply to base permissions */
 	std::vector<permission_overwrite> permission_overwrites;
 
+	/** A set of tags that can be used in a forum channel */
+	std::vector<forum_tag> available_tags;
+
 	/**
-	 * @brief Channel icon (for group DMs)
+	 * @brief The emoji to show as the default reaction button on a forum post.
+	 * Contains either nothing, the id of a guild's custom emoji or the unicode character of the emoji
 	 */
-	utility::iconhash icon;
+	std::variant<std::monostate, snowflake, std::string> default_reaction;
 
 	/**
-	 * @brief Channel banner (boost level locked)
+	 * @brief Channel icon (for group DMs)
 	 */
-	utility::iconhash banner;
```

Comment:
removed banner field. Seems like discord silently removed it. It shouldn't break anyone's code because discord didn't even released an endpoint to request these channel banners kek